app/bundles/LeadBundle/Model/LeadModel.php

```diff
@@ -1906,6 +1922,10 @@ public function import($fields, $data, $owner = null, $list = null, $tags = null
         }
 
         if ($persist) {
+            $lead->setManipulator(new LeadManipulator(
+                'lead',
+                'lead'
+            ));
```

Comment:
Maybe I'm not understanding the task right, but how can you tell if the source of the lead is import over manual creation over UI? This code is called 4 times on 4 different places.

Reply:
It's already long time, so I did not fully recalled what is bundle name and what is object name. Where do you see problem? It's manipulated from lead bundle, lead model so it's lead-lead if I'm right.

I'm not sure if it says something about UI